src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/helper/BulkScanHelper.java

```diff
@@ -72,82 +16,12 @@ public static LocalDate transformFormDateIntoLocalDate(String formFieldName, Str
             .put("A settlement or a transfer of property", "A settlement or a transfer of property")
             .put("Property Adjustment Order", "Property Adjustment Order")
             .build();
-    
+
     public static final Map<String, String> dischargePeriodicalPaymentSubstituteChecklistToCcdFieldNames =
         new ImmutableMap.Builder<String, String>()
             .put("a lump sum order", "Lump Sum Order")
             .put("a property adjustment order", "Property Adjustment Order")
             .put("a pension sharing order", "Pension Sharing Order")
             .put("a pension compensation sharing order", "Pension Compensation Sharing Order")
             .build();
-        
-    public static final Map<String, String> miamExemptionsChecklistToCcdFieldNames =
```

Comment:
I am guessing this is not used anymore right?

Reply:
yep, this was removed from scope

Reply:
and these methods were not called from anywhere.

Reply:
good point